src/components/CompareResults/ResultsMain.tsx

```diff
@@ -17,7 +17,6 @@ function ResultsMain(props: { results: CompareResultsItem[][] }) {
   const styles = {
     container: style({
       backgroundColor: themeColor100,
-      maxWidth: '950px',
```

Comment:
This was clashing with mui's default behavior on `<Container>`; they use `maxWidth: 1200px` for screens bigger than that, and otherwise use a fluid layout.
Because this is the behavior we have generally I decided to keep it, and removed this fixed `maxWidth`.